src/styles/index.scss

```diff
@@ -1,3 +1,68 @@
-body {
+$day-size: 100px;
+$week: 7;
+$gap: 1px;
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body,
+html {
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  display: flex;
+  width: 100%;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
```

Comment:
The gap value should be extracted into a variable since it's a reusable value and is already defined as `$gap` at the top of the file. Use the `$gap` variable here to maintain consistency and ease of maintenance.